go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/docker/go-events v0.0.0-20170721190031-9461782956ad // indirect
 	github.com/docker/go-metrics v0.0.0-20181218153428-b84716841b82 // indirect
 	github.com/docker/go-units v0.3.3
-	github.com/firecracker-microvm/firecracker-go-sdk v0.15.2-0.20190410183546-4f11ac73a965
+	github.com/firecracker-microvm/firecracker-go-sdk v0.15.2-0.20190627223500-b2e8284e890c
```

Comment:
Why is the version of the `v0.17.0` sdk still referenced as `v0.15.2` here? Can we update this to specify `v0.17.0`? Or is that a separate issue that would require changes in the firecracker-go-sdk? I don't remember how this versioning string works in go modules off the top of my head.

Reply:
@sipsma I suspect this is because there is no corresponding tag `v0.17.0` in firecracker-go-sdk. Latest releases there is v0.15.1. This is somewhat confusing.

Reply:
Okay makes sense. That's something we should resolve but doesn't need to be part of this PR.